liveRecorder/scripts/build_ts2mp4_convertor.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+if [ "$#" -lt 2 ]; then
+	echo "usage build_ts2mp4_convertor <ffmpeg build path> <product path>"
+	exit
+fi
+
+FFMPEG_BUILD_PATH=$1
+PRODUCT_ROOT_PATH=$2
+TARGET=ts_to_mp4_convertor
+CONVERTOR_DIR=${PRODUCT_ROOT_PATH}/${TARGET}
+
+[ `uname` = "Linux" ] && OS="linux" || OS="darwin"
+
+export FFMPEG_BUILD_PATH
+export OS
+
+echo "OS=$OS"
+
+
+if [ -e $CONVERTOR_DIR ]; then
+	pushd $CONVERTOR_DIR
+		mkdir -p obj
+		[ -e  ${TARGET} ] && rm -f ${TARGET}
+		[ -e  obj/${TARGET} ] && rm -f obj/${TARGET}
+
+		echo "starting to build ${TARGET}"
+
+		make
+
+		if [ -s  obj/${TARGET} ] ; then
```

Comment:
Instead of checking whether obj/${TARGET}'s size is bigger than zero, check the RC from make with:
```
if [ $? -eq 0 ]
```
Obviously, if make exited with a non zero RC, we failed.